networkx/algorithms/minors.py

```diff
@@ -458,9 +458,7 @@ def blockmodel(G, partition, multigraph=False):
     .. [1] Patrick Doreian, Vladimir Batagelj, and Anuska Ferligoj
            "Generalized Blockmodeling",Cambridge University Press, 2004.
 
-    Notes
-    -----
-    This function has been deprecated. Please use ``quotient_graph`` instead.
+    .. note:: Deprecated in NetworkX v1.11, it is replaced by `quotient_graph``
```

Comment:
This should be

    .. note:: Deprecated in ...

       ``blockmodel`` will be removed in .... Instead, use ``quotient_graph`` with the keyword argument ``relabel=True``.

See http://www.sphinx-doc.org/en/stable/markup/para.html#directive-note for the format of the `note` directive.